pagopa/io-app
# [#172428751] Dispatch email validated event to mixpanel

**Short description:**
When the email is validated, send the event to mixpanel.

**List of changes proposed in this pull request:**
- Added ` ts/sagas/watchProfileEmailValidationChangedSaga.ts` to track the `profileLoadSuccess` and check if the `is_email_validated` change from `false` to `true`.
- Added action `profileEmailValidationChanged` dispatched when the email validation value change with payload: the new value of the `is_email_validated`.
- The analytics middleware now track the action `profileEmailValidationChanged` and dispatch the event `PROFILE_EMAIL_VALIDATION_CHANGED` with payload `{ isEmailValidated: boolean }`to mixpanel. The payload is the new value of the profile `is_email_validated`.

**How to test:**
- Switch from `is_email_validated` from false to true (eg. change the email from the profile section) and verify if the event is received from mixpanel.
